src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/controller/ConfiguredApiController.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.justice.digital.hmpps.digitalprisonreportingmi.controller
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.validation.constraints.Min
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.digitalprisonreportingmi.service.ConfiguredApiService
+
+@Validated
+@RestController
+@Tag(name = "Configured Data API")
+class ConfiguredApiController(val configuredApiService: ConfiguredApiService) {
+
+  @GetMapping("/{reportId}/{dataSetId}/{reportVariantId}")
```

Comment:
In the definition list endpoint, it currently returns just the dataset ID as the resourceName.

Could you update that to return this pattern instead, please? That way the resourceName the UI uses for API calls will match this endpoint.

Reply:
Would this mean to have only the dataSet id as a path parameter and to move the reportId and reportVariantId as query parameters?

Reply:
I mean this line could be updated to match, so the UI uses the right "resource": https://github.com/ministryofjustice/hmpps-digital-prison-reporting-mi/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/service/ReportDefinitionMapper.kt#L48

Reply:
I have now changed this to: `"$productDefinitionId/${report.id}"`

Reply:
And the path of the ConfiguredApiController has also been changed to `"/{reportId}/{reportVariantId}"`